src/frontend/src/lib/utils/token.utils.ts

```diff
@@ -88,3 +88,28 @@ export const calculateTokenUsdBalance = ({
 			})
 		: undefined;
 };
+
+/** Maps a Token object to a TokenUi object, meaning it adds the balance and the USD balance to the token.
+ *
+ * @param token - The given token.
+ * @param $balancesStore - The balances data for the tokens.
+ * @param $exchanges - The exchange rates data for the tokens.
+ * @returns The token UI.
+ */
+export const mapTokenUi = ({
```

Comment:
Should we add a test since it's a utils?

Reply:
i thought of that, but it is actually just a map, in the sense that it actually has no logic inside, just an aggregation of other functions (that have tests). Maybe it is not an util? is there a map folder?

Reply:
If has some logic, it calls `calculateTokenUsdBalance` and map a balance only if the related balance is defined.

Reply:
right! included